src/Widgets/DropZone.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import ShowCaseWidgetProps from '../types/Widgets';
+import { makeid } from '../utils';
+import { addItemToSchema, addItemToUiSchema } from '../context/InteractionMethods';
+import { useVerna } from '../context/VernaContextProvider';
+import type { WidgetProps } from '@rjsf/core';
+
+export default function DropZone({ id }: Partial<WidgetProps>) {
+  const [draggingOver, setDraggingOver] = useState(false);
+  const verna = useVerna();
+
+  function addItem(widgetProps?: ShowCaseWidgetProps) {
+    if (!id) return;
+    const newKey = makeid(10);
+    addItemToSchema(newKey, id, verna, widgetProps?.type);
+    addItemToUiSchema(newKey, id, verna, widgetProps?.widgetName);
+  }
+
+  function onDrop(event: React.DragEvent<HTMLDivElement>) {
+    event.preventDefault();
```

Comment:
Just for my information.

What do you need to use `preventDefault` here?

Reply:
At first, the onDrop was opening the content of my dnd in a new tab (printing its name), but now it doesn't do anything :thinking: 
And here the onDragOver preventDefault makes it work otherwise it doesn't, not really sure why it does act like this tho